agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -11,6 +11,7 @@
 
 #include "wan_monitor.h"
 
+#include <bcl/beerocks_backport.h>
```

Comment:
make_unique is not called in this header, it's called in backhaul_manager.cpp, so the include should be there.